.github/workflows/release.yaml

```diff
@@ -0,0 +1,21 @@
+name: Latest Release
```

Comment:
```suggestion
name: Release
```

Reply:
https://github.com/input-output-hk/cardano-db-sync/pull/86/commits/dd1550ca75a9a8a79c4c6f8e9c3394324e3313fb#diff-499885e3a90e0cf4f87aa4cdc18d4879R1

Using "Post Release" instead, because cardano-node already has a "release" action.